hail/src/main/scala/is/hail/utils/IntervalTree.scala

```diff
@@ -323,6 +333,17 @@ case class IntervalTreeNode[U](i: Interval,
         i.isDisjointFrom(pord, interval) &&
         right.forall(_.isDisjointFrom(pord, interval)))
 
+  def querySingleValue(pord: ExtendedOrdering, position: Any): U = {
```

Comment:
I have an idea for a simple way to remove `RVDPartitioner`s use of `IntervalTree`, which I think will make looking up the partition of a single key even faster than this. But this is a big improvement already, so lets get it in for now.